OpenRA.Game/Game.cs

```diff
@@ -515,6 +515,9 @@ static void RenderTick()
 					}
 				}
 
+				foreach (var player in Game.worldRenderer.World.Players)
+					player.Shroud.ResetChangedCells();
```

Comment:
I can't do the reset until the widgets have drawn, since the radar widget also needs this list.

Unless you just mean calling this as a static method of ShroudRenderer or something?